linux/deploy_vm/templates/post_install_scripts/freebsd_post_config.sh

```diff
@@ -133,6 +133,10 @@ sed -i '' 's/kern.geom.label.gptid.enable="0"/kern.geom.label.gptid.enable="1"/'
 echo 'kern.geom.label.gpt.enable="1"' >>/boot/loader.conf
 echo 'kern.geom.label.ufs.enable="1"' >>/boot/loader.conf
 echo 'kern.geom.label.ufsid.enable="1"' >>/boot/loader.conf
+if [ "$machtype" != "amd64" ] && [ "$machtype" != "x86_64" ]; then
```

Comment:
Is Intel Sapphire Rapids CPU also x86_64?

Reply:
Except amd64 and x86_64, what other types are there?

Reply:
`amd64` and `x86_64` is to identify 64bit guest OS not the CPU model.